site/src/components/Paywall/Paywall.tsx

```diff
@@ -93,12 +125,9 @@ const styles = {
 		margin: 0,
 	},
 	description: (theme) => ({
-		marginTop: 16,
 		fontFamily: "inherit",
-		maxWidth: 420,
-		lineHeight: "160%",
-		color: theme.palette.text.secondary,
-		fontSize: 16,
+		maxWidth: 460,
+		fontSize: 14,
 	}),
 	separator: (theme) => ({
```

Comment:
Making a note for myself: we should update our color theme so that it works better on colored backgrounds. The gray divider color looks really washed out on a saturated colored background

Reply:
Divider thickness doubled to make this more obvious

Before:
<img width="1019" alt="Screenshot 2024-09-09 at 10 20 08 AM" src="https://github.com/user-attachments/assets/3c2b88c3-898b-41c7-b281-91235c1cd76e">

After:
<img width="1019" alt="Screenshot 2024-09-09 at 10 20 14 AM" src="https://github.com/user-attachments/assets/96ae8135-f81d-4552-aded-26baf32ec07c">

Reply:
ooooh, purple divider looks _rad_